doc/ug/setup.tex

```diff
@@ -127,7 +127,7 @@ \section{\texttt{setmd}: Setting global variables.}
 \item[periodicity] (bool[3]) Specifies periodicity for the three
   directions. If the feature PARTIAL_PERIODIC is set, this variable
   can be set to (1,1,1) or (0,0,0) at the moment.  If not it is
```

Comment:
You can use the algorithms without constraints. However particles can then escape your simulation box during the integration which can lead to wrong physics. Which might be unexpected... Therefore I think it is worth mentioning this in the users guide?!